debug.log

```diff
@@ -0,0 +1,6 @@
+[1114/134848.654:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
Do we want to be checking this in?

Reply:
Good call, but I removed it in a subsequent commit